### DISCUSSION TITLE
BSB Specifications

### DISCUSSION THREAD

**Discussion by Ethan-Cmt** [fredlcore/BSB-LAN#666] [Q&A] [answered]
I was wondering if I could find the BSB technical specs somewhere (baudrate, data size, parity, start/stop CRC) and also the compatiblity with RS232 and/or RS485 ?

**Comment by fredlcore** [comment, marked as answer]
The technical documents are linked to in our manual, it is not compatible with RS232/485, the specs are 4800bps, 8O1. For more details, look at the source code.
